example.js

```diff
@@ -0,0 +1,36 @@
+const wayscript = require("wayscript");
+
+
+// Set up http trigger with js
+let data = {"hello": "world"};
+let headers = {"x-my-custom-header": "hello"};
+let status_code = 200;
+let response = wayscript.http_trigger.sendResponse(data, headers, status_code);
+console.log(response)
+
+// Get wayscript context with js
+let event =  wayscript.context.getEvent();
+console.log(event);
+
+
+let process = wayscript.context.getProcess();
+console.log(process);
+
+let processDetail = wayscript.context.getProcessDetailExpandedData();
+console.log(processDetail);
+
+let lairTrigger = wayscript.context.getLairTrigger();
+console.log(lairTrigger);
+
+
+let lair = wayscript.context.getLair();
+console.log(lair);
+
+let workspace = wayscript.context.getWorkspace();
+console.log(workspace);
+
+
+let key = "YOUR APPLICATION KEY"
+// application key can be obtained from workspace settings, or through environmental variable 'WAYSCRIPT_EXECUTION_USER_APPLICATION_KEY'
```

Comment:
Can we add a util method for retrieving the application key, so the user doesn't need to worry about the env var?

Similar to what we have in the python module:
https://github.com/wayscript/wayscript-python/blob/55abcf4f22869fccf6dbc7853444b70339683f2a/src/wayscript/utils.py#L29